src/main/java/de/tum/in/www1/exerciseapp/web/rest/ParticipationResource.java

```diff
@@ -129,22 +128,6 @@ public ParticipationResource(ParticipationService participationService, Particip
             .body(result);
     }
 
-//    /**
-//     * GET  /participations : get all the participations.
-//     *
-//     * @param pageable the pagination information
-//     * @return the ResponseEntity with status 200 (OK) and the list of participations in body
-//     */
-//    @GetMapping("/participations")
-//    @PreAuthorize("hasAnyRole('TA', 'ADMIN')")
-//    @Timed
-//    public ResponseEntity<List<Participation>> getAllParticipations(@ApiParam Pageable pageable) {
-//        log.debug("REST request to get a page of Participations");
-//        Page<Participation> page = participationService.findAll(pageable);
-//        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/participations");
-//        return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
-//    }
-
```

Comment:
why is this code commented out? I guess due to a pagination change?! please add a comment or remove the code completely